### ISSUE TITLE
SCs for users of screen magnifiers

### ISSUE THREAD

**Issue by JAWS-test** [w3c/wcag#1546]
People with visual impairments often use screen magnifiers to perceive content. When I check the accessibility of a web page, e.g. with Zoomtext, I am faced with the problem that there are hardly any WCAG-SCs that intercept frequent problems.

Therefore I want to suggest to introduce these necessary SCs with WCAG 2.3 or 3.0 on the one hand and on the other hand to include examples for screen magnification problems in the Understandings of existing SCs, if they are relevant for the respective SC. So far, many SCs are very focused on screen reader users or on visually impaired people who do not use AT.

Common problems when using screen magnifiers are:

* Distances are too large (e.g. between form field and label, between control element and icon, which shows the status of the control element)
* Anti-aliasing does not work with zoom so that the fonts are displayed roughly pixelated (this is rather a problem for UAAG)
* Focus indicator of the screen magnifier is not displayed or is displayed in the wrong place (e.g., SC 4.1.2 could be supplemented here to the effect that it must also be possible to determine the position of an element programmatically)
* When zooming, the focused element is not displayed in the visible area (cause is often the same as with the wrong display of the focus indicator: problems with CSS positioning of elements)
* Contrasts are not sufficient after color adjustment (however, I cannot imagine any SC formulating requirements that go above SC 1.4.3 and intercept the problem)
* Visible alternative texts for non-text content (SC 1.1.1 does not require alternative texts to be visible)
* Role and status of elements is not correctly visible when using the High Contrast Mode (no problem with the screen magnifier, but also relevant for visually impaired people)
* Page updates that do not receive the focus are not noticeable.

**Comment by alastc** [comment]
Given the overlap with UAAG, and the stage in various processes I'm going to label this with WCAG.next, to pick up in (probably) WCAG 3.0.

If you (or anyone) would like to improve the chances of the items being taken up, a little more information about how to replicate / examples would be useful.
